Dockerfile

```diff
@@ -174,8 +218,8 @@ RUN mkdir -p /tmp/python-home/${USER}/.local/lib/python3.10/ \
   && mv /tmp/python/local/lib/python3.10/dist-packages/ /tmp/python-home/${USER}/.local/lib/python3.10/site-packages/
 
 FROM scratch as production
-COPY --from=apt-dependencies /tmp/apt /
+COPY --from=apt-dependencies /tmp/apt /apt-deps
```

Comment:
That will break users of the image? (all our builds?)

Reply:
These changes here will indeed be a breaking change to the image yeah. As Enrico mentioned [here](https://github.com/aica-technology/control-libraries/issues/140), it would be nicer if this image does not need to have the same ros user configured since we don't need ROS here. That was my motivation for this change.

Reply:
We could move those Python files to `/usr/lib` instead and they there is no assumption on what users are available.

Having everything laid out in one repository simplify the copies downstream and I don't really see what we gain by splitting it in different folders.

To be honest, even the assumption of the user is not that big of a deal because this image makes a lot of assumptions about how it's going to be used anyway.